bids/layout/layout.py

```diff
@@ -110,9 +110,9 @@ class BIDSLayout(object):
         index_associated (bool): Argument passed onto the BIDSValidator;
             ignored if validate = False.
         absolute_paths (bool): If True, queries always return absolute paths.
-            If False, queries return relative paths, unless the root argument
-            was left empty (in which case the root defaults to the file system
-            root).
+            If False, queries return relative paths (for files and directories),
+            unless the root argument was left empty (in which case the root
```

Comment:
I think it's no longer possible for `root` to be empty in a `BIDSLayout`, so we can probably amend the docstring.